e2e/core/viewer/viewer-content-services-component.e2e.ts

```diff
@@ -28,8 +26,10 @@ import { AcsUserModel } from '../../models/ACS/acsUserModel';
 
 import { AlfrescoApiCompatibility as AlfrescoApi } from '@alfresco/js-api';
 import { UploadActions } from '../../actions/ACS/upload.actions';
+import { browser } from 'protractor';
 
 describe('Content Services Viewer', () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 50000;
```

Comment:
AFAIK, this will change the global jasmine timeout.
if these tests require q different timeout, either bigger or smaller, the recommended solution is to same the original timeout, change the value, run the tests, then in afterAll change the timeout to the original value